.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,65 @@
+name: "Bug Report"
```

Comment:
Can we only have the questions mentioned in the bug_report.md file? There are some additional questions in this YAML file.